frontend/src/components/AppSearch.vue

```diff
@@ -30,31 +31,33 @@ export default {
 
   computed: {
     suggestions() {
+      const lower = this.search.toLowerCase()
       return this.courses
-        .filter(item => item.name.match(this.search) || item.code.match(this.search))
+        .filter(item => item.tags.match(lower))
         .slice(0, 5)
     }
   },
 
+  methods: {
+      resetSearch() { this.search = '' }
+  },
+
   created() {
-    fetch('course-names.json')
+    fetch('http://localhost:3000/api/course')
       .then(response => response.json())
       .then(data => {
-        this.courses = data.map((name, id) => ({
-          id,
-          name: name.split(' - ')[1],
-          code: name.split(' - ')[0]
-        }))
+          this.courses = data
       })
+      .catch(err => console.warn(err))
   }
 }
 </script>
 
-<style scoped>
+<style lang='less' scoped>
 
 .search {
     position: relative;
-    font-size: 0.8em;
+    font-size: 0.7em;
```

Comment:
wait i don't get em
it's a proportion of the inherited font size? 0.7em = 70% of parent element's font size?

Reply:
It uses 70% of the font-size of the first parent, grandparent... that has a _set_ font-size.